static/gsApp/types/index.tsx

```diff
@@ -748,15 +748,12 @@ export enum CreditType {
   ERROR = 'error',
   TRANSACTION = 'transaction',
   SPAN = 'span',
-  SPAN_INDEXED = 'spanIndexed',
-  PROFILE_DURATION = 'profileDuration',
-  PROFILE_DURATION_UI = 'profileDurationUI',
+  PROFILE_DURATION = 'profile_duration',
```

Comment:
this was the wrong format the whole time; credit types are serialized in lower case snake case

Reply:
Good catch!

Did this PR accidentally remove some of the credit types (indexed span, monitor seat, uptime), or we don't need those?

Reply:
no, as explained in [this comment ](https://github.com/getsentry/sentry/pull/94663#discussion_r2176067096) i removed all the ones that don't exist on the backend